crates/amaru/src/bin/amaru/cmd/import_ledger_state.rs

```diff
@@ -41,8 +41,8 @@ const BATCH_SIZE: usize = 5000;
 
 static DEFAULT_CERTIFICATE_POINTER: LazyLock<CertificatePointer> =
     LazyLock::new(|| CertificatePointer {
-        transaction_pointer: TransactionPointer {
-            slot: From::from(0),
+        transaction: TransactionPointer {
+            slot: Slot::from(0),
```

Comment:
Do we want to settle on this syntax?

Reply:
For the same reasons I dislike `.into()`, I find `From::from` too opaque in many context. I rather use the target type to make it clearer when reading what we're casting into.

Reply:
(but no strong feelings though. I think this change mainly came from some merge-resolution as I altered this code in anticipation of the PR and went for my version in the end 😅).

Reply:
Actually I find it better too.